project-keeper/src/main/resources/templates/.github/workflows/ci-build.yml

```diff
@@ -110,6 +110,78 @@ jobs:
           path: ${{ steps.build-pk-verify.outputs.release-artifacts }}
           retention-days: 5
 
+      - name: Configure link check
```

Comment:
Is there a way of avoiding the duplicate configuration of github-action-markdown-link-check?

Reply:
Yes, but I didn't want to do this in this PR.